core/src/providers/cohere.rs

```diff
@@ -297,6 +297,16 @@ impl LLM for CohereLLM {
         api_decode(self.api_key.as_ref().unwrap(), tokens).await
     }
 
+    // We return empty string in tokenize to partially support the endpoint.
+    async fn tokenize(&self, text: &str) -> Result<Vec<(usize, String)>> {
```

Comment:
Why do we want this ?

Reply:
I want to be able to use it to compute context size when generating the conversation for the model. I don't need the token strings (which I don't have access to here) but I do need the endpoint.